src/build.sh.in

```diff
@@ -7,22 +7,32 @@ cd gmp-6.2.1
 sed -i -e 's/\(gmp_compile="$cc .*conftest.c\)/\1 $LIBS/g' configure
 sed -i -e 's/\(gmp_compile="$CC .*conftest.c\)/\1 $LIBS/g' configure
 
-if [ "$4" = "false" ]; then
+if [ "$5" = "false" ]; then
     SHARED_LIBRARY_ARG="--disable-shared"
 fi
 
+# $1 => c_compiler
+# $2 => host
+# $3 => target
+# $4 => ocamlc_cflags
+# $5 => supports_shared_libraries
+# $6 => native_c_libraries
+
+echo "ac_cv_func_obstack_vprintf=no ac_cv_func_localeconv=no ./configure --host='$3' --build='$2' CC='$1' CPPFLAGS='$4' LIBS='$6' $SHARED_LIBRARY_ARG"
+echo "make SUBDIRS='mpn mpz mpq mpf' PRINTF_OBJECTS= SCANF_OBJECTS CFLAGS+='-Werror=implicit-function-declaration' CPPFLAGS='$3'"
```

Comment:
This line should move down and the `CPPFLAGS` updated to use the same value as the real command.
Slight suggestion: how about writing
```
(
set -x
... ./configure ...
make ...
)
```
instead, if only to avoid copy-paste mishaps?

Reply:
I deleted the `echo`, I made them just for debug. If you want to add some debugs (which can certainly be useful), can you propose a PR? I will happy to merge it then. But I must do release before.

Reply:
Oh! Sorry, if that was only temporary, this is fine!
Thank you.